lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -4127,42 +4127,25 @@ LLVMToSPIRVBase::transBuiltinToInstWithoutDecoration(Op OC, CallInst *CI,
     // Literal S Literal I Literal rI Literal Q Literal O
 
     Type *ResTy = CI->getType();
-    SPIRVValue *Input;
-    std::vector<Value *> Operands(5);
+
+    auto OpItr = CI->value_op_begin();
+    auto OpEnd = OpItr + CI->getNumArgOperands();
 
     // If return type of intrinsic is greater than 64 then FE moves return
     // variable into arguments list
     if (CI->hasStructRetAttr()) {
       assert(ResTy->isVoidTy() && "Return type is not void");
       ResTy = cast<PointerType>(CI->getOperand(0)->getType())->getElementType();
```

Comment:
May we use OpItr here instead of getOperand?

Reply:
Sure. Changed it in [80b3711](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1213/commits/80b371198f9d197467986749273b905ffeb3dbff)